src/firefly/application.py

```diff
@@ -375,6 +433,10 @@ def setup_runengine_actions(self):
         # Customize some specific actions
         self.queue_autostart_action.setCheckable(True)
         self.queue_autostart_action.setChecked(True)
```

Comment:
Yeah that makes sense. I agree that setting it to checked is a bad idea.

As I think about it, I just got rid of that line entirely ([2bcc2a7](https://github.com/spc-group/haven/pull/226/commits/2bcc2a7550b0021380e391a6ec6df7ad9d52760f)). If you have Firefly open and someone opened a second instance, then suddenly the value of Autostart would change for the first Firefly instance just because the second instance was started. I think it's better that just starting Firefly doesn't alter the state of the queueserver.

Thoughts?

Reply:
Agree!